### ISSUE TITLE
Color contrast issues on website schedule

### ISSUE THREAD

**Issue by erosselli** [djangocon/2025.djangocon.africa#88]
Hello! I wanted to report an accessibility / color contrast issue with some of the text in the website, particularly in the "Our schedule" section of the website homepage. The dates show up in a light grey color that does not have sufficient color contrast with the white background (see attached image). Changing the text color to a darker grey or blue would fix the issue. 

I'm happy to submit a PR to fix it myself, just wanted to call it out here first 😄 

<img width="1303" height="626" alt="Our schedule section of the website, showing the light grey text that does not have sufficient color contrast" src="https://github.com/user-attachments/assets/6d244b8a-a774-4cc0-897a-df620af1aa18" />
